plugin/php/template-parts/blocks/image-card-query.php

```diff
@@ -28,7 +28,8 @@
 if ( has_post_thumbnail( $post_id_image_card ) ) {
 	$thumbnail = get_the_post_thumbnail_url( $post_id_image_card );
 } else {
-	$thumbnail = get_template_directory_uri() . '/assets/images/placeholder.png';
+	global $material_design_plugin;
```

Comment:
❓ **Question**: Don't we have a `get_plugin_instance` for this? I realize it does the same but I think we don't need to explicitly call globals in code

Reply:
Good call, Updated.